nipoppy_new/pyproject.toml

```diff
@@ -0,0 +1,60 @@
+[build-system]
+build-backend = "hatchling.build"
+requires = ["hatchling"]
+
+[project]
+authors = [
+    { name = "Nikhil Bhagwat" },
+    { name = "Michelle Wang" },
+    { name = "Brent McPherson" },
+    { name = "Rémi Gau" },
+]
+classifiers = [
+    "Intended Audience :: Science/Research",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved",
+    "Programming Language :: Python",
+    "Topic :: Software Development",
+    "Topic :: Scientific/Engineering",
+    "Operating System :: POSIX",
+    "Operating System :: Unix",
+]
+dependencies = [
+    "boutiques",
+    "pandas",
+    "pybids",
+    "pydantic",
+    "rich",
+    "rich_argparse",
+]
+description = "Standardized organization and processing of neuroimaging-clinical datasets"
+license = { file = "LICENSE" }
+name = "nipoppy"
+readme = "README.md"
+# TODO requires_python
+version = "1.0.0" # TODO eventually use dynamic versioning
+
+[project.optional-dependencies]
+dev = ["nipoppy[test]", "pre-commit"]
+test = [
+    "pytest",
+    "pytest-mock",
+    "fids@git+https://github.com/neurodatascience/FIDS.git",
```

Comment:
FYI: if you want to make nipoppy accessible via pypi (which I think we should) I think all your dependencies must be installable from pypi or conda or somewhere that is not a github repo.
I am not a 100% sure about this but in any case it would be a good reason to push me publish this thing.